### ISSUE TITLE
Upgrading an item translocator with iron and setting the mode to export crashes the game

### ISSUE THREAD

**Issue by johnnyserbe** [Darkosto/SkyFactory4Issues#501]
## Bug Report

Upgrading an item translocator with iron and setting the mode to export (centre piece points outwards) crashes the game.

## Expected Behaviour

Doesn't crash.

## Steps to Reproduce (for bugs)
https://youtu.be/KtYlhKyV0e0

## Logs

* Client/Server Log: https://gist.github.com/johnnyserbe/57a7d3d4a6c9325a1a96af2ccc46603f
* Crash Log: https://pastebin.com/d84ZwSEe

## World Information

* Preset: Sky Factory 4 (default)

* Prestige: Tested with both enabled and disabled.

* Modpack Version world created in: 4.0.1

* Additional Content Installed: N/A

## Client Information

* Modpack Version: 4.0.2
* Java Version: 1.8.0.191 (64 Bit)
* Launcher Used: JAR Launcher 1.6.91
* Memory Allocated: 4608 MB
* Server/LAN/Single Player: Single Player
* Optifine Installed: No
* Shaders Enabled: No

**Comment by SupaSloth** [comment]
Just had this happen on our server. Full crash unable to restart world.

Details:
v4.0.2
Topography: Sky Factory 4 default
Single-player & Multiplayer

Right-clicking the outer ring of an item translocator while holding an iron ingot fails to open the Filter screen of the translocator. Instead the world crashes. Also tested in single player which causes full client crash.

Tested with other ingots. Iron appears to be the only one with the issue. johnnyserbe's video shows how to replicate perfectly. Crash log from server attached.
[crash-2019-04-27_10.41.51-server.txt](https://github.com/Darkosto/SkyFactory4Issues/files/3124258/crash-2019-04-27_10.41.51-server.txt)

**Comment by Gunner76th** [comment]
Just had this happen in version 4.0.6 version. This is an issue that has been reported and known since Feb 7 of this year also as noted here: https://github.com/TheCBProject/Translocators/issues/59

[crash-2019-06-11_21.14.37-server.txt](https://github.com/Darkosto/SkyFactory4Issues/files/3279304/crash-2019-06-11_21.14.37-server.txt)
This is a copy of our crash report when a user attempted to use the Translocators. Recommendation until the issue is resolved is to disable or remove the translocator mod, as one of the primary reasons for using them causes server crashes.

**Comment by ktole1999** [comment]
so this crash causes world corruption?

**Comment by Gunner76th** [comment]
Yes,  unless you use mcedit to remove the affected block space the world
continually crashes.

**Comment by ktole1999** [comment]
yep just confirmed. When will Translocators be removed pending this resolution? This is a pack breaking issue.

**Comment by Gunner76th** [comment]
Bebo, I actually tested removing/ disabling translocators only to discover
it caused several crafting recipes and are completely unrelated to them to
fail.  My only thought is that the recipe for the translocator items are
being manipulated within the same craft tweaker script as the other recipes
causing a script load failure.  Examples of recipes affected are the resin
recipes for the different tree products.

**Comment by ktole1999** [comment]
Makes sense. Hopefully a workaround can be found. Guess its time for script testing. Thanks for the info



Sent via the Samsung Galaxy S® 6, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by ktole1999** [comment]
looks like he supposedly fixed it last year but has yet to push to Curse. https://github.com/TheCBProject/Translocators/issues/47

**Comment by Gunner76th** [comment]
Is there a released jar that can be downloaded even if it isn't through
curse that can be used by server owners? Or it's it still uncompiled code?

**Comment by ktole1999** [comment]
Yes there is a download link for Jenkins. I wont be able to test for another few hours to see if its actually fixed as mentioned.



Sent via the Samsung Galaxy S® 6, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by Gunner76th** [comment]
I have asked 2 of my administrators to test it locally and see if it works
and if it causes any issues within the pack as a whole such as broken
crafting recipes.  Hopefully I will have some data by tomorrow.
